scripts/actions/abilities/pets/mewing_lullaby.lua

```diff
@@ -0,0 +1,43 @@
+-----------------------------------
+-- Mewing Lullaby
+-- aoe light based sleep and lowers mob TP to zero
+-----------------------------------
+local abilityObject = {}
+
+abilityObject.onAbilityCheck = function(player, target, ability)
+    return 0, 0
+end
+
+abilityObject.onPetAbility = function(target, pet, skill, master, action)
+    local typeEffect = xi.effect.LULLABY
```

Comment:
I know it's called lullaby. However, It was proven that lullaby BRD spells actually aply sleep effect, not lullaby effect, as weird as it sounds.

My point? Either we follow suit OR we confirm in retail the actual effect

Reply:
I'm not 100% knowledgeable on the specifics, and we could apply sleep effect instead of lullaby, but doesn't the chat log explicitly say `lullaby` instead of `sleep`?

Reply:
We can easily use one in the addStatus and the other in the message return, that's not a hurdle. But if you think it actually uses the other status, that's easy to check with a packet cap.

Reply:
Xaver DM'd me about it and that is exactly what we updated it to.